### PR TITLE
Modifi description of "add logout button" image in TutorialSecurity.rst #3256

### PR DESCRIPTION
**Pull request by btkomatsubaratka**
Modifi description of "add logout button" image in TutorialSecurity.rst

Please review #3256

### CODE REVIEW

File: source/Tutorial/TutorialSecurity.rst [hunk 1]
```diff
@@ -912,7 +912,7 @@ JSPからログインユーザーのアカウント情報へアクセス
 
 |
 
-Logoutボタンを押下し、アプリケーションからログアウトする(ログインページが表示される)。
+Logoutボタンを押下すると、アプリケーションからログアウトする(ログインページが表示される)。
```
Comment (yoshikawaa) [inline source/Tutorial/TutorialSecurity.rst]: - [ ] See https://github.com/terasolunaorg/guideline/issues/3256#issuecomment-364346110

Comment (btkomatsubaratka) [reply source/Tutorial/TutorialSecurity.rst]: 8fed7430286b8b2e58a1b6cfbda22aca64255c9e で対応 @btkomatsubaratka

**Comment by btkomatsubaratka** [comment]
@yoshikawaa please check.

**Comment by yoshikawaa** [comment]
@btkomatsubaratka Please rebase for merging.

**Pull request merged**